WalletWasabi/Crypto/Term.cs

```diff
@@ -0,0 +1,17 @@
+using NBitcoin.Secp256k1;
+using WalletWasabi.Crypto.Groups;
+
+namespace WalletWasabi.Crypto
+{
+	public class Term
```

Comment:
fwiw the reason i don't think we'll need it is based on our discussion on slack on whether to use indices instead of allowing some of the generators to be the point at infinity to cancel out unneeded terms of the shared witness on both sides of verification sub-equations that don't need them

Reply:
100% useless. I thought it could be a good idea but it is too much. I will delete this.